src/ontology/cl-edit.owl

```diff
@@ -11263,26 +11267,31 @@ SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0000878 obo:CL_0000864
-SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0000880 obo:CL_0000878)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:ana") Annotation(oboInOwl:hasDbXref "GO_REF:0000031") Annotation(oboInOwl:hasDbXref "PMID:9550136") Annotation(oboInOwl:hasDbXref "PMID:37232741") obo:IAO_0000115 obo:CL_0000880 "A central nervous system macrophage found at the interface between the blood and the cerebrospinal fluid in the brain. This central nervous system macrophage expresses scavenger receptors, has a star-like shaped body.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37232741") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_0000880 "cpMΦ")
+AnnotationAssertion(rdfs:label obo:CL_0000880 "choroid plexus macrophage")
+EquivalentClasses(obo:CL_0000880 ObjectIntersectionOf(obo:CL_4042003 ObjectSomeValuesFrom(obo:RO_0001025 obo:UBERON_0001886) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0005044)))
 
 # Class: obo:CL_0000881 (perivascular macrophage)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "GO_REF:0000031") Annotation(oboInOwl:hasDbXref "PMID:16507898") obo:IAO_0000115 obo:CL_0000881 "A central nervous system macrophage found in small blood vessels in the brain. Markers include CD14+CD16+CD163+.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "GO_REF:0000031") Annotation(oboInOwl:hasDbXref "PMID:16507898") Annotation(oboInOwl:hasDbXref "PMID:37232741") obo:IAO_0000115 obo:CL_0000881 " A central nervous system macrophage that is part of small blood vessels in some brain. Markers include CD14, CD16, CD163. In homeostatic conditions, this central nervous system macrophage has a non-motile cell body with extending and retracting projections through the blood vessel wall.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37232741") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_0000881 "pvMΦ")
 AnnotationAssertion(rdfs:comment obo:CL_0000881 "Markers: Human/monkey, mice: CD163. Human: CD14+CD16+CD163+.")
 AnnotationAssertion(rdfs:label obo:CL_0000881 "perivascular macrophage")
-EquivalentClasses(obo:CL_0000881 ObjectIntersectionOf(obo:CL_0000878 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001981) ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000001483) ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000001889) ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000001925)))
-SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0000881 obo:CL_0000878)
+EquivalentClasses(obo:CL_0000881 ObjectIntersectionOf(obo:CL_4042003 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001981) ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000001483) ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000001889) ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000001925)))
 SubClassOf(obo:CL_0000881 obo:CL_4033054)
+SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0000881 obo:CL_4042003)
```

Comment:
This axiom can be removed, since it is already present in the logical definition.